packages/web-runtime/src/container/versions.ts

```diff
@@ -6,12 +6,12 @@ export const getWebVersion = (): string => {
 }
 
 export const getBackendVersion = ({ store }: { store: Store<unknown> }): string => {
-  const backendVersion = store.getters.user.version
-  if (!backendVersion || !backendVersion.string) {
+  const backendStatus = store.getters.capabilities?.core?.status
+  if (!backendStatus || !backendStatus.versionstring) {
     return undefined
   }
-  const product = backendVersion.product || 'ownCloud'
-  const version = backendVersion.string
-  const edition = backendVersion.edition
+  const product = backendStatus.product || 'ownCloud'
+  const version = backendStatus.productversion || backendStatus.versionstring
+  const edition = backendStatus.edition
   return `${product} ${version} ${edition}`
```

Comment:
Just had a random shower thought on whether we should insert `(development)` here if process.env.NODE_ENV === development

Reply:
To detect whether the bundle-watched dist is being used or the shipped `web` in oCIS - I've been adding a lot of console.log's recently to debug caching

Reply:
oha, nice! good idea :-)

Reply:
will do that in a followup to unblock the next ocis beta